danfojs-browser/src/plotting/plot.js

```diff
@@ -411,7 +411,7 @@ export class Plot {
         }
       });
 
-      trace['x'] = this.ndframe[this_config['y']].values;
+      trace['x'] = this.ndframe[this_config["x"]].values;
```

Comment:
@adithyaakrishna Thanks for the fix! One small comment before I merge, use a single quote inline with the existing style.

Reply:
My bad, Instead of single quotes, I gave double :/
Will fix this ASAP